### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by Darthagnon**
Previous pull request didn't work?
https://github.com/Barre/privaxy/pull/22
Issue #21

### CODE REVIEW

**Review by Barre — CHANGES_REQUESTED** [review]
Thank you for this PR. I'd prefer to not endorse directly in the README any particular filter set. Otherwise, looks ok to me.

**Comment by Darthagnon** [comment]
> particular filter set

Is that the reference to filterlists.com?

**Comment by Darthagnon** [comment]
Hmmm... that's a point - are custom filterlists currently supported?

File: README.md [hunk 1]
```diff
@@ -22,7 +22,7 @@ This is an early release without authentication on the management API. You shoul
 
 ### Using a pre-built binary
 
-Pre-built binaries for Macos and Linux (x86_64) are provided on [github releases](https://github.com/Barre/privaxy/releases).
+Pre-built binaries for MacOS, Linux (x86_64) and Windows are provided on [Github releases](https://github.com/Barre/privaxy/releases).
```
Comment (ThinkChaos) [inline README.md:25]: The (not so) new spelling is "macOS":
```suggestion
Pre-built binaries for macOS, Linux (x86_64) and Windows are provided on [Github releases](https://github.com/Barre/privaxy/releases).
```

File: README.md [hunk 2]
```diff
@@ -39,11 +39,15 @@ Pre-built binaries for Macos and Linux (x86_64) are provided on [github releases
 
 1. Navigate to the web gui at `http://127.0.0.1:8000`, click on "Download CA certificate".
 2. Install the downloaded certificate locally.
-    - Macos: <https://support.apple.com/guide/keychain-access/add-certificates-to-a-keychain-kyca2431/mac>
-    - Linux: `cp privaxy_ca_certificate.pem /usr/local/share/ca-certificates/`
+   - MacOS: https://support.apple.com/guide/keychain-access/add-certificates-to-a-keychain-kyca2431/mac
```
Comment (ThinkChaos) [inline README.md:42]:
```suggestion
   - macOS: https://support.apple.com/guide/keychain-access/add-certificates-to-a-keychain-kyca2431/mac
```

File: README.md [hunk 3]
```diff
@@ -39,11 +39,15 @@ Pre-built binaries for Macos and Linux (x86_64) are provided on [github releases
...
+   - Linux: `cp privaxy_ca_certificate.pem /usr/local/share/ca-certificates/`
+   - Windows CLI: Open an Admin Command Prompt, then run `certutil –addstore –f "Root" privaxy_ca_certificate.pem` ([Source](https://superuser.com/questions/1031444/importing-pem-certificates-on-windows-7-on-the-command-line))
+   - Windows GUI: `Control Panel` >> `Network and Sharing Center` >> `Internet Options` (link, bottom left) >> `Content` tab >> `Certificates` button >> `Trusted Root Certification Authorities` tab >> `Import` button >> `Next`, `Browse` to the `privaxy_ca_certificate.pem`, `Next`, `Finish`. **Done**, you can close it all.
 3. Configure your local system to pass http traffic through privaxy.
-   - Macos: <https://support.apple.com/guide/mac-help/change-proxy-settings-network-preferences-mac-mchlp2591/mac>
-   - Ubuntu (gnome): <https://phoenixnap.com/kb/ubuntu-proxy-settings>
+   - MacOS: https://support.apple.com/guide/mac-help/change-proxy-settings-network-preferences-mac-mchlp2591/mac
```
Comment (ThinkChaos) [inline README.md:47]:
```suggestion
   - macOS: https://support.apple.com/guide/mac-help/change-proxy-settings-network-preferences-mac-mchlp2591/mac
```

**Pull request closed without merging**